components/sections/HowItWorksSection.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import content from "@/app/content";
+
+export function HowItWorksSection() {
+  return (
+    <section id="how-it-works" className="w-full py-12 md:py-24 lg:py-32 bg-background">
+      <div className="container px-4 md:px-6">
+        <div className="flex flex-col items-center justify-center space-y-4 text-center mb-12">
+          <div className="space-y-2">
+            <div className="inline-block rounded-lg bg-muted px-3 py-1 text-sm">
+              {content.howItWorks.badge}
+            </div>
+            <h2 className="text-3xl font-bold tracking-tighter sm:text-5xl">
+              {content.howItWorks.title}
+            </h2>
+            <p className="max-w-[900px] text-muted-foreground md:text-xl/relaxed lg:text-base/relaxed xl:text-xl/relaxed">
+              {content.howItWorks.description}
+            </p>
+          </div>
+        </div>
+        <div className="mx-auto grid max-w-5xl items-start gap-8 sm:grid-cols-2 md:gap-12 lg:grid-cols-3">
+          {content.howItWorks.steps.map((step, index) => (
+            <div key={index} className="grid gap-1 text-center">
```

Comment:
Try to improve the naming on keys to reduce collisions

```suggestion
            <div key={`home-steps-${index}`} className="grid gap-1 text-center">
```